addons/web_editor/static/src/js/editor/odoo-editor/test/spec/link.test.js

```diff
@@ -6,14 +6,11 @@ import {
     click,
     deleteBackward,
     insertText,
-    insertParagraphBreak,
     insertLineBreak,
     testEditor,
-    createLink,
```

Comment:
You can remove the definition of the `createLink` function from `web_editor/static/src/js/editor/odoo-editor/test/utils.js` too, since it's no longer used anywhere.

Reply:
Sure! any other changes ?